libcontainer/configs/mount_unsupported.go

```diff
@@ -0,0 +1,34 @@
+//go:build !linux
+// +build !linux
+
+package configs
+
+type Mount struct {
```

Comment:
Should we just make this an empty struct like what was done with [cgroups](https://github.com/opencontainers/runc/blob/da548c1c1b0c43cd8718636e8acfdb536e010bbe/libcontainer/configs/cgroup_unsupported.go#L9) and [namespaces](https://github.com/opencontainers/runc/blob/da548c1c1b0c43cd8718636e8acfdb536e010bbe/libcontainer/configs/namespaces_unsupported.go#L8). We don't need anything from the structure right? It's just because libcontainer/configs is imported at all that's halting compilation

Reply:
Yeah. Theoretically the other fields should be fine, but since I'm not directly consuming the struct I'm fine with this as well.

Reply:
I really like this, given that we can't just embed it